netci.groovy

```diff
@@ -229,7 +109,7 @@ build(params + [WCF_LINUX_BUILD_NUMBER: linuxBuildJob.build.number],
 // **************************
 
 // Define build string
-def codeCoverageBuildString = '''build.cmd /p:Coverage=true /p:WithCategories=OuterLoop'''
+def codeCoverageBuildString = '''build.cmd /p:OSGroup=Windows /p:Coverage=true /p:WithCategories=\"\\\"InnerLoop;OuterLoop\\\"\"'''
 
```

Comment:
Have you tried running code coverage runs like this using both Inner and Outer loop tests?